sympy/core/relational.py

```diff
@@ -186,6 +190,10 @@ def _eval_simplify(self, ratio, measure):
             if v is not None:
                 r = r.func._eval_relation(v, S.Zero)
 
+            numer, denom = dif.as_numer_denom()
+            if numer.is_zero is False and numer.is_number is True:
```

Comment:
This is saying `n/d op 0` is the same as `n op 0` which is only true if you can multiply both sides of `op` by `d`. If `d` is negative and `op` is an inequality then `op` changes. Example: `x/y < 0` where `x = Symbol('x', positive=True)`, `y = Symbol('y', negative=True)`.